src/app/containers/PageHandlers/withVariant/index.test.jsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
+import { useParams, useLocation } from 'react-router-dom';
+import WithVariant from '.';
+import { frontpageRegexPath } from '#app/routes/regex';
+
+jest.mock('react-router-dom', () => ({
```

Comment:
is it possible to not mock this and use something like https://www.npmjs.com/package/history?

Reply:
Fixed!

Reply:
You can actually completely remove this mock by changing your tests to be like:

```js
  describe('service with no default variant', () => {
    it('should not redirect', () => {
      const history = createMemoryHistory();
      const service = 'news';
      const match = getMatchProps('news');

      history.push(`/${service}`, { service }); 

      render(
        <Router history={history}>
          <WithVariantHOC match={match} />
        </Router>,
      );
      expect(history.location.pathname).toEqual('/news');
    });
  });
```

more on this here https://github.com/ReactTraining/history/blob/3f69f9e07b0a739419704cffc3b3563133281548/docs/GettingStarted.md#basic-usage

Reply:
Fixed!